src/obfl-specification.html

```diff
@@ -1854,8 +1897,7 @@ <h4 id="L9514" class="include">on-toc-start</h4>
     </dd>
   <dt>Variables</dt>
     <dd>
-      <p>In the use-when condition, the commonly available variables as
-        listed with the evaluate element can be used.</p>
+      <p>The use of variables is not supported in the use-when condition.</p>
```

Comment:
This is just weird. I mean, I know there isn't an actual use-case at the moment, but saying that the use of variables is not supported is basically the same as saying you can't use the `use-when` attribute (if there are no variables involved the expression resolves to a constant value), and therefore confusing. It's less confusing to just leave out the "Variables" section completely in this case. Or better, add `$volumes`. I can vaguely imagine a use case where you say something at the beginning about how the TOC is structured, but not if there is only one volume.